### PR TITLE
Cherry-pick GP commits #1193 to #1299

### PR DESCRIPTION
**Pull request by MatthewFagan**

### CODE REVIEW

**Comment by pavan-tri** [comment]
This has chart changes and we need a corresponding chart change as well

File: components/dashboard/src/OauthClientApproval.tsx [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+        <div id="oauth-section" className="flex-grow flex w-full">
+            <div id="oauth-section-column" className="flex-grow max-w-2xl flex flex-col h-100 mx-auto">
+                <div className="flex-grow h-100 flex flex-row items-center justify-center" >
+                    <div className="rounded-xl px-10 py-10 mx-auto">
+                        <div className="mx-auto pb-8">
+                            <img src={gitpodIcon} className="h-16 mx-auto" />
+                        </div>
+                        <div className="mx-auto text-center pb-8 space-y-2">
+                            <h1 className="text-3xl">Authorize {clientName}</h1>
+                        <h4>You are about to authorize {clientName} to access your Gitpod account including data for all workspaces.</h4>
```
Comment (pavan-tri) [inline components/dashboard/src/OauthClientApproval.tsx:35]: Branding fix @MatthewFagan ?    (process.env.REACT_APP_HOSTNAME) ?

**Review by pavan-tri — APPROVED** [review]
Need to check on EE components. we need to disable in install

File: components/dashboard/src/OauthClientApproval.tsx (hunk 1 above)
Comment (MatthewFagan) [reply components/dashboard/src/OauthClientApproval.tsx]: OK, fixed

**Pull request merged**